pkgs/development/python-modules/open-clip-torch/default.nix

```diff
@@ -1,43 +1,50 @@
 {
   lib,
-  stdenv,
   buildPythonPackage,
-  braceexpand,
+  fetchFromGitHub,
+
+  # build-system
+  pdm-backend,
+  setuptools,
+
+  # dependencies
   ftfy,
   huggingface-hub,
-  pandas,
   protobuf,
-  pytestCheckHook,
   regex,
   sentencepiece,
   timm,
   torch,
   torchvision,
   tqdm,
+
+  # checks
+  pytestCheckHook,
+  braceexpand,
+  pandas,
   transformers,
-  setuptools,
   webdataset,
-  wheel,
-  fetchFromGitHub,
+
+  stdenv,
 }:
 buildPythonPackage rec {
   pname = "open-clip-torch";
-  version = "2.24.0";
+  version = "2.26.1";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "mlfoundations";
     repo = "open_clip";
     rev = "refs/tags/v${version}";
-    hash = "sha256-ugbXnXiOY9FrNvr8ZxnAgZO/SLCVoXbRgupi8cUwflU=";
+    hash = "sha256-XjPOsGet8VNzwEwzz14f1nF3XOgpkb4OERIc6VrDDZ8=";
   };
 
-  nativeBuildInputs = [
+  build-system = [
+    pdm-backend
     setuptools
```

Comment:
```suggestion
```
I think we no longer need `setuptools`, either.

Reply:
Indeed, done.